src/sentry/api/serializers/models/alert_rule_trigger.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+
+from collections import defaultdict
+
+import six
+
+from sentry.api.serializers import register, Serializer
+from sentry.incidents.models import AlertRuleTrigger, AlertRuleTriggerExclusion
+
+
+@register(AlertRuleTrigger)
+class AlertRuleTriggerSerializer(Serializer):
+    def serialize(self, obj, attrs, user):
+        return {
+            "id": six.text_type(obj.id),
+            "alertRuleId": six.text_type(obj.alert_rule_id),
+            "label": obj.label,
+            "thresholdType": obj.threshold_type,
+            "alertThreshold": obj.alert_threshold,
+            "resolveThreshold": obj.resolve_threshold,
```

Comment:
Just to confirm, we're going to keep timeWindow on the rule level?

Reply:
Yeah, since it's a core part of the subscription from snuba. Otherwise we'd end up needing a subscription per trigger.